contrib/upstart/cilium-net-daemon.conf

```diff
@@ -1,2 +1,2 @@
 env PATH=/usr/local/clang/bin:/usr/local/sbin:/usr/local/bin:/usr/bin:/usr/sbin:/sbin:/bin
-exec cilium -D daemon run -d eth1
+exec cilium -D daemon run --ipv4 --lb
```

Comment:
Yes but the LB functionality is enabled regardless of the `LBMode` value. I thought we added `LBMode` to signal the use case of a standalone LB without additional forwarding to local containers, i.e. listening on a `net_device` and then perform LB if configured.

It is completely pointless if a user has to enable `--lb` to use LB functionality on a regular node. I see that the k8s service integration currently depends on `--lb`. That must be changed, see `daemon/daemon.go:120`. Providing `--lb` is in conflict with direct routing mode.